scripts/protogen.sh

```diff
@@ -44,9 +44,9 @@ rm codegen/protocol-test-codegen-local/build/smithyprojections/protocol-test-cod
 rm codegen/protocol-test-codegen-local/build/smithyprojections/protocol-test-codegen-local/RPCEventStream/swift-codegen/Package.swift
 rm codegen/protocol-test-codegen-local/build/smithyprojections/protocol-test-codegen-local/Waiters/swift-codegen/Package.swift
 
-# Regenerate the Package.swift with protocol tests included and services excluded
+# Regenerate the Package.swift with services & runtime tests excluded
 cd AWSSDKSwiftCLI
-swift run AWSSDKSwiftCLI generate-package-manifest --include-protocol-tests --exclude-aws-services --exclude-runtime-tests ..
+swift run AWSSDKSwiftCLI generate-package-manifest --exclude-aws-services --exclude-runtime-tests ..
```

Comment:
I don't think we need this step for regenerating Package.swift at all anymore.  None of the AWS service clients are used in protocol tests

Reply:
Deleted as suggested.